clean_build.sh

```diff
@@ -4,26 +4,23 @@ cd "$(dirname "$0")" || exit 1
 
 # some environments have cmake v2 as 'cmake' and v3 as 'cmake3'
 # check for cmake3 first then fallback to just cmake
-
-B_CMAKE=$(command -v cmake3 2>/dev/null)
-if [ "$?" -eq 0 ]; then
-    continue
-else
-    if command -v cmake 2>/dev/null; then
-        B_CMAKE=$(command -v cmake)
-    else
-        echo "ERROR: CMake not in $PATH, cannot build! Please install CMake, or if this persists, file a bug report."
-        exit 1
-    fi
+[ -n "$B_CMAKE" ] || B_CMAKE=$(command -v cmake3)
+[ -n "$B_CMAKE" ] || B_CMAKE=$(command -v cmake2)
```

Comment:
`cmake2` will probably refer to cmake v2.x if present. This won't work, because the project has `cmake_minimum_required (VERSION 3.4). I agree with the change of checking style otherwise.

Reply:
Fixed!